microsoft-edge/webview2/concepts/enterprise.md

```diff
@@ -0,0 +1,36 @@
+---
+description: Understand how to manage WebView2 applications
+title: Managing WebView2 applications
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/15/2020
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html, enterprise, group policy, manageability 
+---
+
+# Managing WebView2 applications  
+
+[WebView2][WebView2Landing] is a component that developers use to build their applications, and the developers may deploy a [self-updating Evergreen WebView2 Runtime][Webview2ConceptsDistributionUnderstandRuntimeInstallerPreview] on user device to power their applications. This document discusses how IT admins may manage WebView2 applications and Runtime. Feedback from both IT admins and developers are welcomed on [WebView2 Feedback repo][GithubMicrosoftedgeWebviewfeddback].  
+
+## Group policies for WebView2
+
+IT admins may use group policy objects (GPO) to configure policy settings for WebView2. The following set of policies are/are not applicable to WebView2,
```

Comment:
```suggestion
IT admins may use Group Policy Objects (GPO) to configure policy settings for WebView2. The following list describes any policy differences that apply to WebView2.
```

Reply:
Will leave as-is. I'd rather start this as what policies WV2 has, rather than how it is different from the browser.